core/controllers/pages.py

```diff
@@ -71,19 +71,26 @@ def get(self):
                 self.redirect(target_url)
                 return
 
-
-class AboutPage(base.BaseHandler):
-    """Page with information about Oppia."""
+class CreditsPage(base.BaseHandler):
```

Comment:
Here and elsewhere, there should be two blank lines above each top-level class defn.